app/jobs/scheduled/code_review_sync_repos.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Jobs
+  class CodeReviewSyncRepos < ::Jobs::Scheduled
+    every 1.hour
+
+    def execute(args = {})
+      DiscourseCodeReview::GithubRepoCategory.pluck(:name, :repo_id).each do |repo_name, repo_id|
+        ::Jobs.enqueue(
+          :code_review_sync_commits,
+          repo_name: repo_name,
+          repo_id: repo_id,
```

Comment:
In the `code_review_sync_commits`, we only check for the type of the `repo_name` parameter. But we should also add a check for the `repo_id` argument as well.

Reply:
In this plugin, this thing is different than usually. `repo_name` is the "primary key", not `repo_id`. That's why `repo_id` is always an optional parameter in the plugin.